javascript/javascript2/week2/moviesEx.js

```diff
@@ -0,0 +1,84 @@
+// i used just few objects of the array here
+const movies = [{"title": "'71","year": 2014,"rating": 7.2,"votes": 41702,"running_times": 5940},{"title": "'A' gai wak","year": 1983,"rating": 7.4,"votes": 11942,"running_times": 6300},{"title": "'Breaker' Morant","year": 1980,"rating": 7.9,"votes": 10702,"running_times": 6420},{"title": "'Crocodile' Dundee II","year": 1988,"rating": 5.5,"votes": 47180,"running_times": 6480},{"title": "(500) Days of Summer","year": 2009,"rating": 7.7,"votes": 412368,"running_times": 5700},{"title": "*batteries not included","year": 1987,"rating": 6.6,"votes": 25636,"running_times": 6360},{"title": "...E tu vivrai nel terrore! L'aldilà","year": 1981,"rating": 6.9,"votes": 16484,"running_times": 5220}];
+
+// Working with movies
+
+//Movie with shortest title // Extra
+const filter = movies.map(movie => movie.title.length);
+//console.log(filter);
+const shortestTitle = filter.reduce((sum, current) => (sum < current) ? sum : current);
+const shortestTitleObj = movies.find(item => item.title.length == shortestTitle);
+//console.log(shortestTitleObj);
+//console.log(shortestTitle);
+
+//Movie with longest title // Extra
+const longestTitle = filter.reduce((sum, current) => (sum > current) ? sum : current);
+//console.log(longestTitle);
+
+// 1. //movies with a short title
+const shortTitleMovies = movies.filter(title => (title.title.length >= 1 && title.title.length <= 2) ? true : false);
+//console.log(shortTitleMovies);
+
+// 2. //Movies with long title 
+const longTitleMovies = movies.filter(title => (title.title.length >= 25) ? true : false);
+//console.log(longTitleMovies);
+
+// 3. //number of movies made between 1980-1989
+const yearOfMadeMovies = movies.filter(year => (year.year >= 1980 && year.year <= 1989) ? true : false);
```

Comment:
This will end up being a boolean (true, false), so you do not need the ternary statement.
`(year.year >= 1980 && year.year <= 1989)`

Also, `year.year` is confusing to read. Call it something like `movie`.

```suggestion
const yearOfMadeMovies = movies.filter(movie => (movie.year >= 1980 && movie.year <= 1989));
```